package.json

```diff
@@ -31,6 +31,7 @@
     "test:types": "tsc -p packages/test-utils/types && tsc -p packages/server-test-utils/types"
   },
   "dependencies": {
+    "@vue/composition-api": "^0.3.2",
```

Comment:
I believe this should be in devDependencies because it's only used during testing.

Reply:
I've placed it in `devDependencies`, but I think there's a lot that can go in there (`devDependencies` is being introduced in this PR for the `package.json`)

Reply:
Good point. I need to clean that up